.travis.yml

```diff
@@ -14,43 +14,32 @@ cache:
     - ".eslintcache"
 
 stages:
- - basic
- - advanced
- - versions
+  - basic
+  - advanced
 
 matrix:
   include:
     - os: linux
       node_js: "10"
-      env: NO_WATCH_TESTS=1 JEST="--maxWorkers=2 --cacheDirectory .jest-cache" JOB_PART=basic
+      env:
+        NO_WATCH_TESTS=1 JEST="--maxWorkers=2 --cacheDirectory .jest-cache" JOB_PART=basic
```

Comment:
Also you can use multiline yaml array here (like I did in my pr)

Reply:
yeah, makes sense. For this PR I only run prettier on the file.